examples/ssx-test-nextauth-email/package.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Unused dependencies
* mysql
* nodemailer
Unused devDependencies
* @types/node
* @types/nodemailer
* dotenv

Reply:
Removed mysql and dotenv, but @types/node nodemailer and @types/nodemailer are required